aptos-move/aptos-vm-environment/src/environment.rs

```diff
@@ -59,7 +59,7 @@ impl AptosEnvironment {
     /// block executor where this optimization is needed. Note: whether the optimization will be
     /// enabled or not depends on the feature flag.
     pub fn new_with_delayed_field_optimization_enabled(state_view: &impl StateView) -> Self {
-        let env = Environment::new(state_view, true, None).try_enable_delayed_field_optimization();
+        let env = Environment::new(state_view, false, None).try_enable_delayed_field_optimization();
```

Comment:
try_enable fails quiently? or why is it try?

Reply:
Only if the feature flag is enabled